examples/src/main/java/org/apache/spark/examples/ml/JavaQuantileDiscretizerExample.java

```diff
@@ -54,14 +54,15 @@ public static void main(String[] args) {
     });
 
     Dataset<Row> df = spark.createDataFrame(data, schema);
-
-    // Note that we compute exact quantiles here by setting `relativeError` to 0 for
-    // illustrative purposes, however in most cases the default parameter value should suffice
+    // $example off$
+    // Output of QuantileDiscretizer for such small datasets differ wrt underlying cores.
```

Comment:
Can we say "... small datasets can be dependent on number of partitions" instead - strictly speaking it is not the # cores but the # partitions of the dataset.